fractal_tasks_core/tasks/yokogawa_to_ome_zarr.py

```diff
@@ -145,7 +145,7 @@ def yokogawa_to_ome_zarr(
 
     # Read useful information from ROI table and .zattrs
     adata = read_zarr(f"{zarrurl}/tables/FOV_ROI_table")
-    pxl_size = extract_zyx_pixel_sizes(f"{zarrurl}/.zattrs")
+    pxl_size = ngff_image_meta.get_pixel_sizes_zyx(level=0)
```

Comment:
Does it make sense to take this and the next part up to line 116 with the rest of the metadata extraction? Let's discuss whether this is placed better :)

Reply:
Fixed with https://github.com/fractal-analytics-platform/fractal-tasks-core/pull/528/commits/7d453684e7ae07b43b8431c2df6691050797f94a.